app/controllers/home_controller.rb

```diff
@@ -2,13 +2,29 @@ class HomeController < ApplicationController
   before_action :authenticate_client!
 
   def index
+    @products = Product.page(params[:page]).per(10)
```

Comment:
We shouldn't be accessing products in a controller called `HomeController`. Let's rename it to `ProductsController`

Reply:
I've covered this on this commit [9375356](https://github.com/AguirreCampiNicolas/onboarding/pull/1/commits/937535640e112d2f9e0a82c57177df8c0228c24c)